internal/accounts/get.go

```diff
@@ -0,0 +1,70 @@
+/*
+ * Flow CLI
+ *
+ * Copyright 2019-2021 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package accounts
+
+import (
+	"github.com/onflow/flow-cli/internal/command"
+	"github.com/onflow/flow-cli/pkg/flow"
+	"github.com/onflow/flow-cli/pkg/flow/services"
+	"github.com/psiemens/sconfig"
+	"github.com/spf13/cobra"
+)
+
+type flagsGet struct {
+	Code bool `default:"false" flag:"code" info:"Display code deployed to the account"`
+}
+
+type cmdGet struct {
+	cmd   *cobra.Command
+	flags flagsGet
+}
+
+func NewGetCmd() command.Command {
+	return &cmdGet{
+		cmd: &cobra.Command{
+			Use:     "get <address>",
+			Short:   "Gets an account by address",
+			Aliases: []string{"fetch", "g"},
```

Comment:
I think it'd be best to focus on getting the new refactor out before adding additional aliases, just in case we decided we don't like them or want to change the command names altogether.

Reply:
Yes, for sure. This was there before. I will remove